platform-ui-skin/src/main/webapp/skin/less/vuetify/lib/vuetify.less

```diff
@@ -27428,7 +27428,7 @@ html.overflow-y-hidden {
 .v-snack--absolute {
   height: 100%;
   position: absolute;
-  z-index: 1;
+  z-index: 1000;
```

Comment:
Please pay attention to not change a very generic class for a specific use case. It will make a lot of regressions in fact.
To fix this, either use newly introduced re-usable component https://community.exoplatform.com/portal/dw/activity?id=413406 (https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/common/components/Notifications.vue#L112-L130) or define this specific CSS attribute for Documents only.

Reply:
> Please pay attention to not change a very generic class for a specific use case. It will make a lot of regressions in fact. To fix this, either use newly introduced re-usable component https://community.exoplatform.com/portal/dw/activity?id=413406 (https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/vue-apps/common/components/Notifications.vue#L112-L130) or define this specific CSS attribute for Documents only.

ok thanks @boubaker , this PR will be closed.